pkg/sink/codec/debezium/codec.go

```diff
@@ -68,13 +76,31 @@ func (c *dbzCodec) writeDebeziumFieldSchema(
 	switch col.GetType() {
 	case mysql.TypeBit:
 		n := ft.GetFlen()
+		var v uint64
+		var err error
+		if col.GetDefaultValue() != nil {
+			val, ok := col.GetDefaultValue().(string)
+			if !ok {
+				log.Error(
```

Comment:
May be we don't need to print error info here? we return the error info to outside, and maybe outside have print the error already. I'm not sure, you can check again